native/cocos/bindings/jswrapper/v8/ScriptEngine.h

```diff
@@ -322,8 +322,8 @@ class ScriptEngine final {
      * This function asks ScriptEngine to interrupt lengthy JavaScript operations and run the provided callback, passing the supplied data to it.
      * Once the callback completes, control returns to the JavaScript code. Multiple interrupt requests may be ongoing.
      * This function can be called from a different thread without needing a Locker. The registered callback should not reenter the interrupted Isolate.
-     * @param callback
-     * @param data
+     * @param callback the callback to be called when interrupt happens.
```

Comment:
when `interruption` happens.

`interrupt` is a verb.

Reply:
fixed